plugins/typescript/src/generators/generateReactQueryComponents.test.ts

```diff
@@ -231,7 +231,7 @@ describe("generateReactQueryComponents", () => {
       /**
        * Get all the pets
        */
-      export const useListPets = (variables: ListPetsVariables, options?: Omit<reactQuery.UseQueryOptions<ListPetsResponse, ListPetsError, ListPetsResponse>, \\"queryKey\\" | \\"queryFn\\">) => { const { fetcherOptions, queryOptions, queryKeyFn } = usePetstoreContext(options); return reactQuery.useQuery<ListPetsResponse, ListPetsError, ListPetsResponse>(queryKeyFn({ path: \\"/pets\\", operationId: \\"listPets\\", variables }), () => fetchListPets({ ...fetcherOptions, ...variables }), {
+      export const useListPets = <TSelectData = ListPetsResponse>(variables: ListPetsVariables, options?: Omit<reactQuery.UseQueryOptions<ListPetsResponse, ListPetsError, TSelectData>, \\"queryKey\\" | \\"queryFn\\">): reactQuery.UseQueryResult<TSelectData, ListPetsError> => { const { fetcherOptions, queryOptions, queryKeyFn } = usePetstoreContext(options); return reactQuery.useQuery<ListPetsResponse, ListPetsError, TSelectData>(queryKeyFn({ path: \\"/pets\\", operationId: \\"listPets\\", variables }), () => fetchListPets({ ...fetcherOptions, ...variables }), {
```

Comment:
I think we should call this generic `TData` instead of `TSelectData`, so it's consistant with react-query:

![image](https://user-images.githubusercontent.com/1761469/187516653-af218775-9eb4-484e-acca-9ed3495067c8.png)

Reply:
Agree! 👍